nextflow_schema.json

```diff
@@ -221,7 +221,7 @@
             "properties": {
                 "expdesign": {
                     "type": "string",
-                    "description": "A tab-separated experimental design file in OpenMS' own format (TODO link). All input files need to be present as a row with exactly the same names. If no design is given, unrelated, unfractionated runs are assumed.",
+                    "description": "A tab-separated experimental design file in OpenMS' own [format](http://www.openms.de/doxygen/release/latest/html/classOpenMS_1_1ExperimentalDesign.html#details). All input files need to be present as a row with exactly the same names. If no design is given, unrelated, unfractionated runs are assumed.",
```

Comment:
There was even a TODO for it :+1: 

Cool. The OpenMS documentation looks good.